core/src/clients/git.rs

```diff
@@ -628,6 +629,13 @@ impl Git {
             bail!("No arguments provided to git command");
         }
 
+        let mut sys = System::new();
+        sys.refresh_processes();
```

Comment:
In the past we've found that this call can be exorbitantly expensive - up to 20 seconds even for a PC doing normal amounts of work. If you peep `process.rs`, we've found a way to filter in a way that makes this a bit cheaper:

```rs
 let mut system = sysinfo::System::new();
 let refresh_kind =
        sysinfo::ProcessRefreshKind::new().with_exe(sysinfo::UpdateKind::OnlyIfNotSet);
 system.refresh_processes_specifics(refresh_kind);